README.md

```diff
@@ -372,6 +372,11 @@ frequently-changing representations that can't reasonably be fetched from the
 source and inconvenient to reify as changes in the domain (typically for storage
 reasons).
 
+`<target>` (optional) If present, the event will only be delivered to the
+subscriber with the given name.  
+The use case for this is when performing an "initial sync" (as described above)
+and we intend to avoid delivering these `noop` events to any other subscriber.
```

Comment:
I am bit puzzled if this means you are advocating the usage of `noop` + `target` or `noop` or `whatever event type` + `target`. Honestly I agree in general with the direction this is headed but I wonder if there should a new event type like `sync` instead. Right now I don't think any of the apps cares about `noop` and in fact when I read about `noop` is really a noop, i.e., nothing happened ignore. Even more weird is having an event for `create` and `update` but given the nature of routemaster where very easily you can be hammered by a ton load of events. Most of times the apps do the same (upsert) behaviour regardless it is a create or an update.

Reply:
Agreed.
Looks like so far we only have `upsert`, `delete` and `sync` behaviour everywhere.